### ISSUE TITLE
ubuntu-mate and ubuntu-xfce do not show desktop

### ISSUE THREAD

**Issue by jpmunson** [linuxserver/docker-webtop#48]
I am trying to run the ubuntu-mate version of the webtop image. When I do, and load localhost:3000, I just get the black dot in a white screen. The Mate desktop does not show. 

[docker-compose-orig-ubuntu-mate.txt](https://github.com/linuxserver/docker-webtop/files/8824828/docker-compose-orig-ubuntu-mate.txt)

[Dockerlog.txt](https://github.com/linuxserver/docker-webtop/files/8824848/Dockerlog.txt)

**Comment by decatoncheir** [comment]
@jpmunson You may try to start the docker-compose with an empty config volume (in your case, empty directory "/Users/munsonj/efs/home/gmail.com/codework.instructor.1").

In my case, I tried "latest" image and then switch to "ubuntu-mate" with the same volume mounted to config volume, the scripts and config files created by "latest" image may be in trouble with new image. After a start the "ubuntu-mate" image with an empty config volume, all things work normally.

**Comment by jpmunson** [comment]
Yes, that solved it, thanks! But the way I want to use such a container is to mount a user's files from an NFS mount, in which case what I'm mapping to /config will generally not be empty. This characteristic is probably related to the creation of all those folders like "Documents", "Music", "Video", etc, which are not useful in my use case. Do you know where that happens? Thanks.

**Comment by decatoncheir** [comment]
@jpmunson you can keep the config volume if you use the same distribution and the same desktop manager. 

By removing the config files and scripts related to X and desktop manager before image change should do the trick.

**Comment by huapox** [comment]
> try this: 
> 
> `docker run -it --rm -e L=zh_CN --net=host infrastlabs/docker-headless:cmate`

**Comment by aptalca** [comment]
@huapox 

Please don't suggest running commands for random 3rd party images

**Comment by Niko-La** [comment]
so @jpmunson for every time we switch from `webtop:ubuntu-mate' to 'webtop:ubuntu-xfce' or 'webtop:ubuntu-kde'
we must clear the mounted config folder?

for me `mate` is the only one that show up in ubuntu.

**Comment by beckeradm** [comment]
I got a similar same issue. I pulled the latest ubuntu-KDE branch, build the image, and ran the container in a brand-new folder. The login works just fine, but I only got a tiny terminal window instead of the KDE desktop. right click opens some kind of OB-Menu.

git clone --branch ubuntu-kde  https://github.com/linuxserver/docker-webtop.git
docker build -t XXX .
docker run -it -p 4000:3000 XXX 

Did I miss a step?
